src/Adapter/ParameterContainer.php

```diff
@@ -79,7 +84,15 @@ public function offsetExists($name)
      */
     public function offsetGet($name)
     {
-        return (isset($this->data[$name])) ? $this->data[$name] : null;
+        if (isset($this->data[$name])) {
+            return $this->data[$name];
+        }
+
+        if (isset($this->nameMapping[$name])) {
+            return $this->data[$this->nameMapping[$name]];
```

Comment:
not sure if this return statement is valid, can you explain please?

Reply:
Since now parameters names are no longer used internally in the parametercontainer, the only keys we have are now `c_0`, `c_1`, while previousely, we can also have `foo` or `bar`.
As far as I remember, it was required to fix https://travis-ci.org/github/galette/laminas-db/jobs/685493304#L414